### DISCUSSION TITLE
Is it possible to limit globalFilterFn to some columns?

### DISCUSSION THREAD

**Discussion by kobe651jp** [TanStack/table#5299] [General]
globalFilterFn search all columns by defaults.

[official sample](https://github.com/TanStack/table/blob/main/examples/react/filters/src/main.tsx)

In my case I need omit some columns like createdAt etc,It's is possible change globalFilterFn behavior？

This is for performance reasons and to eliminate unnecessary noise.

**Comment by TonyVeigel** [comment]
https://tanstack.com/table/v8/docs/api/features/filters#getcanglobalfilter
